packages/connect-deeplink/src/index.ts

```diff
@@ -174,7 +175,7 @@ export class TrezorConnectDeeplink implements ConnectFactoryDependencies {
     }
 
     private buildUrl(method: string, params: any, callback: string) {
-        return `${this.schema}://connect?method=${method}&params=${encodeURIComponent(
+        return `${this._settings.deeplinkUrl}?method=${method}&params=${encodeURIComponent(
```

Comment:
I am having here some changes related to this https://github.com/trezor/trezor-suite/pull/14490/files#diff-161cfff841ef61661da9b9f9b576e2f9d4d96db4dbc697b6202506404d5acdcdR4

But we can adjust once we are ready to merge it.